index.html

```diff
@@ -278,6 +278,23 @@ <h2 class="title-block__name _title">Work experience</h2>
                 </div>
                 <div class="work-card__main">
                   <p class="work-card__text">
```

Comment:
We should use "p" node only for text, not for block. Html structure should by like that:
```html
<div class="work-card__main">
 <h2 class="work-card__h2">Title</h2>
 <p class="work-card__text">
  Text...
 </p>
 ...
</div>
```